app/assets/javascripts/discourse/app/components/topic-timeline/container.hbs

```diff
@@ -108,10 +108,10 @@
       <a
         class="now-date"
         onClick={{this.updatePercentage}}
-        title={{this.nowDate}}
+        title={{age-with-tooltip this.nowDate this.nowDateOptions}}
```

Comment:
`age-with-tooltip` outputs HTML, right? Do we really want to use that as the `title` of an `<a` tag? 🤔

Reply:
I think we could drop the title here completely as we have

```
<span class="relative-date" title="Mar 13, 2023 4:21 pm" data-time="1678742467682" data-format="tiny">18h ago</span>
```

on the actual date... would should suffice for screen readers etc.